.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,7 +6,7 @@
 - [ ] Passed code style checking (`make lint`)
 - [ ] Changes are complete (i.e. I finished coding on this PR)
 - [ ] All changes have test coverage
-- [ ] For user-facing API changes, API doc string has been updated.
+- [ ] For user-facing API changes, API doc string has been updated. For new C++ functions in header files, their functionalities and arguments are well-documented. 
 - [ ] To my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change
 
 ### Changes ###
```

Comment:
Let's also fix the typo below.  Intersting -> Interesting

Reply:
Cool fixed that.